src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/TestHost.cs

```diff
@@ -93,9 +136,14 @@ public byte[] GetPeerCert(bool exportAsPem)
             }
         }
 
-        public string GetFQDN()
+        public Stream Ping()
         {
-            return Dns.GetHostEntry("127.0.0.1").HostName;
+            return new MemoryStream(Encoding.UTF8.GetBytes("Service has started"));
+        }
+
+        public Stream State()
+        {
+            return new MemoryStream();
```

Comment:
This is just scaffolded in so that we can provide server state at a future date - such as NTLM available - as a JSON response.

We can either have a JSON response with all the required information (ping once) or individual endpoints like NtlmSupported, KerberosSupported, IsWindows to determine service state. I prefer the "ping once" approach where we obtain all relevant server state once.